service/src/java/org/apache/hive/service/cli/thrift/ThriftHttpCLIService.java

```diff
@@ -136,6 +136,12 @@ public void onClosed(Connection connection) {
               ConfVars.HIVE_SERVER2_SSL_KEYSTORE_PATH.varname 
               + " Not configured for SSL connection");
         }
+        String keyStoreType = hiveConf.getVar(ConfVars.HIVE_SERVER2_SSL_KEYSTORE_TYPE).trim();
+        if (keyStoreType.isEmpty()) {
+          keyStoreType = KeyStore.getDefaultType();
+        }
+        String keyStoreAlgorithm = hiveConf.getVar(ConfVars.HIVE_SERVER2_SSL_KEYSTORE_ALGORITHM).trim();
```

Comment:
`keyStoreAlgorithm` looks unused? Might have to be used around line 153?

Reply:
I will add

Reply:
For the MetastoreConf changes in testing because the previous HiveConf metastore related properties are all deprecated, I just switch to the corresponding properties in MetastoreConf